### DISCUSSION TITLE
Views on updating the default values of certain customizations

### DISCUSSION THREAD

**Discussion by vedang** [vedang/pdf-tools#133] [Announcements]
As I prepare for a 1.0.0 version of `pdf-tools`, I am going through all the default values of the variables. There are certain variables that I'm thinking of changing the default values for, and I would love community feedback on it. 

The most controversial of these will probably be `pdf-view-use-scaling`. This variable controls how PDFs look on HiDPI screens (Retina Display, and certain newer monitors). The default value of this variable is false and people invariably get stuck with PDFs rendering badly because they don't realise they have to set it to true on HiDPI screens. I'm thinking of changing the default value to `t`. This will mean existing users may have to change this value to `nil` in their configurations, but it is a future-facing change meant to make it easier for new users / new monitors.

I'd like to know what the community thinks of this change and whether I should go ahead and make it.

**Comment by Hugo-Heagren** [comment]
I for one agree with the change, for the reaons you outline.
